sed/core/processor.py

```diff
@@ -178,10 +178,35 @@ def __repr__(self):
             df_str = "Data Frame: No Data loaded"
         else:
             df_str = self._dataframe.__repr__()
-        attributes_str = f"Metadata: {self._attributes.metadata}"
-        pretty_str = df_str + "\n" + attributes_str
+        pretty_str = df_str + "\n" + "Metadata: " + "\n" + self._attributes.__repr__()
         return pretty_str
 
+    def _repr_html_(self):
+        html = "<div>"
+
+        html += (
+            f"<details><summary>Dataframe</summary>{self.dataframe.head()._repr_html_()}</details>"
+        )
+
+        # Add expandable section for dataframe
+        html += f"<details><summary>Dask</summary>{self.dataframe._repr_html_()}</details>"
+
```

Comment:
I somehow find this redundant information to the first part. I'd suggest to remove the "Dataframe" section, and move this code there.